server/src/server.ts

```diff
@@ -0,0 +1,45 @@
+import express, { Request, Response, NextFunction } from 'express'
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+import user from './routes/user'
+const { database, connectionString, port } = require('./dotenv.config')
+
+const app = express()
+
+app.use(express.urlencoded({ extended: false }))
+app.use(express.json())
+
+dotenv.config()
+
+mongoose
```

Comment:
What node version are you using? Maybe try with top level await instead of .then .catch